### ISSUE TITLE
Workshop Title doesn't show on website

### ISSUE THREAD

**Issue by arockenberger** [uio-carpentry/2020-01-31-R#2]
Looks like there is something wrong with the repository.

The title has been set correctly to "R for Reproducible Scientific Analysis" in _config.yml but it doesn't show in the grey box on the [website](https://uio-carpentry.github.io/2020-01-31-R/).

I thought it might be because the title originally was much longer and the length caused a problem, but it wasn't the case.

I don't know what the problem might be, perhaps someone with more experience with Jekyll can try to find out?

**Comment by annesbe** [comment]
I have absolutely no idea how to fix this. I realise now that what we did in the previous course was simply to change the venue in the index file to the workshop name, and this seems to make it work, so for now, that's what I'll do.

**Comment by arockenberger** [comment]
>… what we did in the previous course was simply to change the venue in the index file to the workshop name, and this seems to make it work, so for now, that's what I'll do.

PLEASE DON'T!

It will mess with the automatic harvesting of data from the Carpentries AMY Database!

**Comment by arockenberger** [comment]
It's not so problematic - better to find out WHY this happens. No stress!

**Comment by annesbe** [comment]
changed it back, but I have absolutely no clue why the title won't show up. Every single file looks identical to the python workshop where it looks ok.

**Comment by arockenberger** [comment]
There could be something wrong with the template itself. E.g. setting "R" als the "flavour" or something like this could trigger the bug.
We could file an Issue in the original repository and have them look into it!?

**Comment by annesbe** [comment]
yeah, it is the template. It does not include a title at the top either, only the venue.

**Comment by arockenberger** [comment]
Question: Did you import the repository from (https://github.com/carpentries/workshop-template) or did you import a past repository from uio-carpentry?

**Comment by arockenberger** [comment]
The reason I'm asking is that we (Naoe and I) have seen a similar problem with some of our past websites; it could be an erroneous repo here on uio-carpentry from which this problem is inherited.

**Comment by annesbe** [comment]
I tried both. The last thing I did was to try with the original template from the carpentries, but it has no title on the front page. The previous course website was created by Tobias, and there the venue is simply set to the workshop title instead, which is why we didn't notice that it didn't actually have a title either.

**Comment by arockenberger** [comment]
Ok: Which files have you been editing when setting up the website?
_config.yml (in the main folder)
index.md (in the main folder)

any other?

**Comment by annesbe** [comment]
Yeah, inside _includes/sc/ I also edited schedule.html and syllabus.html to match our version of the workshop

**Comment by arockenberger** [comment]
That should be fine, though. Hm…

**Comment by lexnederbragt** [comment]
I would file an issue at https://github.com/carpentries/workshop-template, as far as I can tell you are doing everything right.

**Comment by arockenberger** [comment]
Done that! (https://github.com/carpentries/workshop-template/issues/641)

**Comment by fmichonneau** [comment]
Hi, it looks like you're currently using an old (more than 1 year old) version of the workshop template. Adding a title in the jumbotron is a new feature, so you'll need to create the website by importing the latest version of the template.

You might be able to create a new branch, pull the latest version, and do a pull request on your repo. There might be a few merge conflicts.


Something like: 

```
git checkout -b use-latest-template
git pull https://github.com/carpentries/workshop-template.git gh-pages
git push origin use-latest-template
```

go to GitHub and open a PR against your repository.

**Comment by arockenberger** [comment]
Thank you, @fmichonneau !
@annesbe Do you want to try that or is it best to create a new repo and delete the old one?

**Comment by annesbe** [comment]
Thank you @fmichonneau, I will make a note of it and make sure to use the new template next time. 

@arockenberger I'll just create a new repo and delete the old one. There is not much I need to change in the template to adapt it to this workshop.
